AUR packaging

I'll make a PKGBUILD soon so that it can be uploaded.

How's:
```
# Maintainer: Mario Ortiz Manero <omitted here>
# Maintainer: Oliver Ford <omitted here>
# shellcheck disable=SC2034,SC2154
pkgname=polybar-pulseaudio-control
pkgver=1.3
pkgrel=1
pkgdesc='Polybar module to control PulseAudio'
url='https://github.com/marioortizmanero/polybar-pulseaudio-control'
license=('MIT')
source=("${url}/archive/v${pkgver}.tar.gz")
md5sums=('e6693e35cb023fd1c4a02dc4ddc6b765')
arch=('any')
depends=(
    'bash'
    'libpulse'
    'pulseaudio'
)
optdepends=(
)
conflicts=(
)

package() {
    set -e
    cd "$pkgname-$pkgver"

    install -D -m644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"

    mkdir -p "$pkgdir/usr/bin"
    install -D -m755 "./pulseaudio-control.bash" "$pkgdir/usr/bin/$pkgname"
}
```

--- a/README.md
+++ b/README.md
@@ -11,7 +11,20 @@ A feature-full volume control module for PulseAudio. Also known as Pavolume. Mai
 ![example](screenshots/example.png)
 
 
-## Dependencies
+## Installation
+
+### Arch
+
+Install [`pulseaudio-control`](https://aur.archlinux.org/packages/pulseaudio-control/) from the AUR with your preferred method, for example:
+```
+$ yay -S pulseaudio-control
+```
+
+### Other Linux
+
+Download the [bash script](https://github.com/marioortizmanero/polybar-pulseaudio-control/blob/master/pulseaudio-control.bash) from this repository, or extract it from [the latest release](https://github.com/marioortizmanero/polybar-pulseaudio-control/releases/latest), and put it somewhere on your `$PATH`.
+
+#### Dependencies
 
 [`pulseaudio`](https://www.freedesktop.org/wiki/Software/PulseAudio/) with `pactl` and `pacmd` in your `$PATH`. You might want to have [`pavucontrol`](https://freedesktop.org/software/pulseaudio/pavucontrol/) installed to easily control pulseaudio with a GUI. The script can send notifications if enabled, for which you'll need a notification daemon like [`dunst`](https://github.com/dunst-project/dunst).
 

